### ISSUE TITLE
[CTW/16v16v16v16] Road Trip to Sunset Town

### ISSUE THREAD

**Issue by master00ofdsiware** [OvercastCommunity/CommunityMaps#3]
### Checklist
Check what applies to you. *Add an X between the brackets or click the checkboxes when you have submitted the issue.*
- [X] I have [pruned](https://pgm.dev/docs/guides/packaging/pruning-chunks) the map
- [X] I have agreed with assigning the CC BY-SA license to this map, as mentioned in the README
- [X] I have created an XML file
- [X] I have created an map image
- [X] I have uploaded the map zip file to a file sharing service
- [X] The map has been tested locally to make sure it works

# Map Name
Road Trip to Sunset Town

## Gamemode
CTW

## Team Sizes
4 teams, 16 players each

## Screenshots
https://imgur.com/a/ti247c0

# XML
Included in the zip.

# Map Image
Included in the zip.

# Download Link
[Road Trip to Sunset Town.zip](https://github.com/OvercastCommunity/CommunityMaps/files/5032727/Road.Trip.to.Sunset.Town.zip)

**Comment by CoWinkKeyDinkInc** [comment]
It reminds me a lot of like Bamboo Valley :)

**Comment by master00ofdsiware** [comment]
`Also provide a small description of the map and how you envision it playing.`

This is a 4 team CTW in the shape of a circle, with player spawns on the outside of the circle, and all the wool rooms in a single building at the map's center. This is visible in the overhead screenshot. Of course, there is a large amount of barriers at the center to prevent players from jumping between wool rooms.

The map is fairly traversible; the extensive tree coverage allows players to rarely touch the ground when moving from side to side. A defense strategy would be to burn or chop the trees down. My main concern is the difficulty of getting the wool from the team directly opposite your own, since that requires going near an unrelated team's spawn. If gameplay tests show this to be a significant issue, I am considering adding some sort of bypass route so that players trying to capture the opposite wool have less to deal with on the way. I don't believe defense is currently too easy, considering the two non-adjacent wool lanes.

The XML should work perfectly; I have tested it locally.

**Comment by fixitqixit** [comment]
This map is awesome. Can't wait to play it.

**Comment by Pugzy** [comment]
### Packaging
- The download contains 10 map files, are all of these in use? I could only find `Wool Spawn Point` and `Circular Highway` and one for each teams wool room in game.

### XML
XML is fine just a few gaps between some of the region definitions, nothing to cry about as the rest seems to work fine.

### Map
- In the wool rooms teams are able to pass items (through the glass) to other players. Is this intentional?
- The team sizes seem a little large for the map, we can see how it plays but something to note.

**Comment by master00ofdsiware** [comment]
> The download contains 10 map files, are all of these in use? I could only find `Wool Spawn Point` and `Circular Highway` and one for each teams wool room in game.

Yes, all the map files are used. The remaining ones are found in the observer spawn.

> In the wool rooms teams are able to pass items (through the glass) to other players. Is this intentional?

It's not strictly intentional, but it could make gameplay slightly more interesting. I think this "feature" can be kept unless it proves to be problematic.
> The team sizes seem a little large for the map, we can see how it plays but something to note.

I also noticed that when making the XML. If tests show that 64 players make the map overcrowded, the capacity can be reduced to 48 or thereabouts.

**Comment by Pugzy** [comment]
Just had a match on this and the match lasted sub 5 mins. 

- Had to explain to players how it works as the shared wool room concept is a little confusing.
- Way too many trees could be reduced to 90% of what there is now as the majority of the lanes are trees, people just block spamming on top of them rather than walking ground level.
- The lanes to the wool room in mid are also very cluttered, buildings on both sides all the way down.
- Not sure where from but players found TNT which made the terrain around the water a nightmare, not sure TNT is needed.
- Felt cramped when played with only 12 per team, could be because of the trees and thin lanes.
- Monument location confusing as you spawn looking away from it and is then hidden by the waterfalls.

**Comment by master00ofdsiware** [comment]
Thanks for the recap; I will modify the map to play better and be a bit less confusing in certain aspects.

**Comment by fixitqixit** [comment]
Just to reiterate some of Pugzy's above points, the lanes are indeed too thin and many people shared the same feeling after playing the map a few minutes ago. Heres a few of my own suggestions:

- I love having tnt in the map but once the lanes get blown up all the water flows all over the place and the lanes become even more difficult to navigate. I suggest either lowering the explosion radius of tnt or reducing the amount of water on the map
- Just in general the lanes should be widened everywhere

I like this map, im looking forward to future updates.
